tests/generation/test_utils.py

```diff
@@ -1457,6 +1457,7 @@ def test_contrastive_generate_dict_outputs_use_cache(self):
             for output in (output_contrastive, output_generate):
                 self._check_outputs(output, input_ids, model.config, use_cache=True)
 
+    @slow  # TODO(Joao): remove this. Some models (e.g. data2vec, xcom, roberta) have an error rate between 1 and 10%.
```

Comment:
Just wondering, since this test uses batch size 1 and short sequence, could we run it 10 times and check similar to #22996 ?